src/Worlds/DebugWorld/DebugWorld_PlumChain.layer

```diff
@@ -70,32 +70,32 @@ GenericEntity PlumTrader {
     Flags 1027
    }
    PlumTraderTestItems {
-    coords -0.627 0.918 0.343
```

Comment:
Not part of this PR

Reply:
Had to be updated because the models have a different origin now